packages/demux/src/AloxideActionHandler.ts

```diff
@@ -78,6 +80,20 @@ export class AloxideActionHandler extends AbstractActionHandler {
       .then<void>(() => {});
   }
 
+  private async rollbackIndexState(blockNum: number): Promise<void> {
+    this.indexStateModel.blockNumber = blockNum;
+    this.indexStateModel.blockHash = '';
+
+    return this.dataAdapter
+      .update(this.getIndexStateModelName(), this.indexStateModel, {
+        entity: this.indexStateEntity,
+      })
+      .catch(err => {
+        this.log.error('---- demux updateIndexState error:', err);
+      })
+      .then<void>(() => {});
```

Comment:
What's its purpose?

Reply:
it update last processed block in db to roll back block number. After rollback, demux can continue reading from that block num.

Reply:
I was only curious about this dumb function :)

` .then<void>(() => {});`

Reply:
updated